pubspec.yaml

```diff
@@ -1,12 +1,12 @@
 name: pana
 description: PAckage aNAlyzer - produce a report summarizing the health and quality of a Dart package.
-version: 0.22.1
+version: 0.22.2-wip
 repository: https://github.com/dart-lang/pana
 topics:
   - tool
 
 environment:
-  sdk: '>=3.0.0 <4.0.0'
+  sdk: ^3.2.0
```

Comment:
Do we have any pressing need for this? Let's not force-upgrade people just for the sake of upgrading.

Reply:
That's fair. Not necessary for now :)